src/ui/pages/container/CL-Transcription/ReviewAudioTranscriptionLandingPage.jsx

```diff
@@ -1423,7 +1423,8 @@ useEffect(() => {
               <Box sx={{mb:2,}} >
                 <Tabs value={tabValue} onChange={handleTabChange} aria-label="user-tabs">
                     <Tab label="L1 & L2 Transcription" sx={{ fontSize: 17, fontWeight: '700', marginRight: '28px !important' }} />
-                    {ProjectDetails?.metadata_json?.acoustic_enabled_stage >=2 || ProjectDetails?.project_stage >=2 && 
+                    {ProjectDetails?.metadata_json?.acoustic_enabled_stage >=2  && 
```

Comment:
@Pursottam6003 
should this not be `{ProjectDetails?.metadata_json?.acoustic_enabled_stage <=2 &&` ?